docs/FAQ.md

```diff
@@ -109,10 +109,41 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I deal with divergent changes ('??' after the ID)?
+
+A [divergent change][glossary_divergent_change] represents a change that has two
+or more visible commits associated with it. To refer to such commits, you must
+use their [commit ID][glossary_commit_id]. Most commonly, the way to resolve
+this is to abandon the unneeded commits (using `jj abandon <commit ID>`). If you
+would like to keep both revisions of the change, you can `jj duplicate` one of
+them before abandoning it.
+
+Usually, the different commits associated with the divergent change ID should all
+appear in the log, but due to #2476, they may not. If that happens, you can
+either use `jj log -r 'all()' | grep <change id>` or disable the
+`revsets.short-prefixes` config option.
+
+### How do I deal with conflicted branches ('?' after branch name)?
+
+A [conflicted branch][branches_conflicts] is a branch that refers to multiple
+different commits that can't be resolved automatically. Resolving conflicted
+branches is usually done by setting the branch to the correct commit using
+`jj branch set <commit ID>`.
+
+Usually, the different commits associated with the conflicted branch should all
+appear in the log, but if they don't you can use `jj branch list | grep -A 2 <branch>`
+to show all the commits associated with it.
```

Comment:
`| grep A 2 <branch>` is not guaranteed to include all targets. Maybe this is still a good-enough approximation for now. We should probably allow `jj branch list <branch>`.

Reply:
Yeah, I can change it to a larger number, I suppose, but `jj branch list <branch>` would be really good (and/or `jj branch list --conflicted`).

Reply:
(Optional) I'd just say "Use `jj branch list` to see all the commits associated with a branch". The `grep` trick is nice, but this doesn't seem like the right place for it.

> We should probably allow jj branch list <branch>.

+1.

Reply:
Fair enough, let me remove the grep, people can figure it out on their own.